doc/controls/LoadingView.md

```diff
@@ -11,6 +11,7 @@ Source|ILoadable|Gets and sets the source `ILoadable` associated with this contr
 LoadingContent|object|Gets or sets the content to be displayed during loading/waiting.
 LoadingContentTemplate|DataTemplate|Gets or sets the template to be used to display the LoadingContent during loading/waiting.
 LoadingContentTemplateSelector|DataTemplateSelector|Gets or sets the template selector to be used to display the LoadingContent during loading/waiting.
+DisableAnimantions|bool|Gets and sets whether animations will run when transitioning between states.
```

Comment:
```suggestion
DisableAnimations|bool|Gets and sets whether animations will run when transitioning between states.
```

Reply:
maybe `UseTransition` would be better suited?